pyproject.toml

```
@@ -0,0 +1,88 @@
+[tool.poetry]
+name = "sample-size"
+version = "0.0.1a1"
+description = "A python module implementing power analysis to estimate sample size"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+statsmodels = "0.13.0"
+
+[tool.poetry.dev-dependencies]
+flake8 = "^3.7"
+coverage = "^5.4"
+isort = "^5.6.3"
+black = "^21.7b0"
+mypy = "^0.910"
+pytest = "^6.2.2"
+pytest-sugar = "^0.9.4"
+pytest-cov = "^2.11.1"
+
+[tool.poetry.scripts]
+qa = "poetry_scripts:qa"
+test = "poetry_scripts:test"
+format = "poetry_scripts:format_fix"
+format_check = "poetry_scripts:format_check"
```

nit: I feel like CLI usually use hyphen instead of underscores for some reason?
```suggestion
format-check = "poetry_scripts:format_check"
```

---

sure, although that is actually follow godaddy example :D